app/views/posts/new/_post_form.html.erb

```diff
@@ -6,7 +6,6 @@
                     required: true,
                     minlength: 5,
                     maxlength: 100 %>
-  <%= f.hidden_field :branch, :value => @branch %>
```

Comment:
Not sure why this line is here ?! its causing problems because a Post doesn't have a branch column